apps/client/src/components/Questions.js

```diff
@@ -280,7 +292,10 @@ const Questions = ({
             key={`question-${q.id}-${index}`}
           >
             <QuestionAnswer
-              disabled={!!checker.isConclusive()}
+              disabled={
+                !!checker.isConclusive() ||
```

Comment:
It returns a bool, no need to do double negation.

Reply:
Good point!